_install-and-configure/additional-plugins/ingest-attachment-plugin.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: default
+title: Ingest-attachment plugin
+parent: Installing plugins
+nav_order: 20
+
+---
+
+# Ingest-attachment plugin
+
+The `ingest-attachment` plugin enables OpenSearch to extract content and other
+information from files using the Apache text extraction library
+[Tika](https://tika.apache.org/).
+Supported document formats include PPT, PDF, RTF, ODF and many more.
```

Comment:
Is there any way to link to tika documentation which lists out supported file formats?

Reply:
You're right, that is the better approach. I skipped it just out of concerns, as it is dependent on the tika version used here: https://github.com/opensearch-project/OpenSearch/blob/main/plugins/ingest-attachment/build.gradle
Currently this is 2.9.2
I will link to supported formats here: https://tika.apache.org/2.9.2/formats.html

But it might be necessary to keep this from time to time updated, especially, once newer versions of tika are imported - like the upcoming version 3.0.0 (currently beta)